apps/client/src/components/EnvironmentForm.vue

```diff
@@ -224,7 +228,7 @@ watch(localEnvironment.value, () => {
       required
       :disabled="!props.isEditable || !props.canManage"
       :options="stageOptions"
-      @update:model-value="resetCluster()"
+      @update:model-value="resetCluster(); setQuotaOptions()"
```

Comment:
Pas sur que de passer 2 functions au model-value soit une bonne idée.

Peut-être qu'on peut refactor pour n'avoir qu'une seule fonction, no ?

Reply:
```suggestion
      @update:model-value="() => { resetCluster(); setQuotaOptions() }"
```
Comme ceci @fabienChaillou ?